doc/md/getting-started.md

```diff
@@ -37,11 +37,14 @@ go mod init <project>
 
 ## Create Your First Schema
 
+- The full example exists in [GitHub](https://github.com/ent/ent/tree/master/examples/start).
```

Comment:
Why it's here?

Reply:
Some people like to see the example before they walk through it (like me for example)